console/src/components/Services/Data/DataActions.js

```diff
@@ -689,6 +693,7 @@ const makeMigrationCall = (
                 successMsg,
                 errorMsg,
                 shouldSkipSchemaReload,
+                false,
```

Comment:
@beerose Could you please add this change for the very next update?
Related to https://github.com/hasura/graphql-engine/issues/5436